test/caf_error_stop_test.f90

```diff
@@ -11,30 +12,44 @@ function test_prif_this_image() result(tests)
 
         tests = describe( &
                 "A program that executes the prif_error_stop function", &
-                [ it("exits with a non-zero exitstat when stop code is an integer", check_integer_stop_code) &
-                 ,it("exits with a non-zero exitstat when stop code is an character", check_character_stop_code) &
+                [ it("exits with a non-zero exitstat when the program omits the stop code", exit_with_no_stop_code) &
+                 ,it("prints a character stop code and exits with a non-zero exitstat", exit_with_character_stop_code) &
+                 ,it("prints an integer stop code and exits with exitstat equal to the stop code", exit_with_integer_stop_code) &
                 ])
     end function
 
-    function check_integer_stop_code() result(result_)
+    function exit_with_no_stop_code() result(result_)
         type(result_t) :: result_
         integer exit_status
 
         call execute_command_line( &
-          command = "./build/run-fpm.sh run --example error_stop_integer_code > /dev/null 2>&1", &
+          command = "./build/run-fpm.sh run --example error_stop_with_no_code > /dev/null 2>&1", &
           wait = .true., &
           exitstat = exit_status &
         )   
         result_ = assert_that(exit_status /= 0)
 
     end function
 
-    function check_character_stop_code() result(result_)
+    function exit_with_integer_stop_code() result(result_)
         type(result_t) :: result_
         integer exit_status
 
         call execute_command_line( &
-          command = "./build/run-fpm.sh run --example error_stop_character_code > /dev/null 2>&1", &
+          command = "./build/run-fpm.sh run --example error_with_stop_integer_code > /dev/null 2>&1", &
```

Comment:
The name of the example program is still wrong, so it is a problem that the tests are passing, because they should theoretically fail? Because fpm should complain that there is no such example?